src/components/GoogleCalendarButton.js

```diff
@@ -26,41 +26,90 @@ export default function GoogleCalendarButton() {
     Mo: '2020-04-06T',
   };
   const events = [];
-  const handleClick = () => {
-    sections.forEach(
-      (section) => {
-        // '2015-05-28T09:00:00-07:00'
-        const dow = section.event.dow;
-        const date = dateOfTheFirstClass[dow];
-        const startTime = `${date + section.event.start.hour}:${section.event.start.minute}:00${timeZone}`;
-        const endTime = `${date + section.event.end.hour}:${section.event.end.minute}:00${timeZone}`;
-        const event = {
-          summary: `${section.subjectId}${section.courseId}`,
-          description: section.name,
-          location: section.event.location,
-          start: {
-            dateTime: startTime,
-            timeZone: 'America/Chicago',
-          },
-          end: {
-            dateTime: endTime,
-            timeZone: 'America/Chicago',
+  const clientID = '4436581685-mcdkhnh50a6di21k15do7d4a2orq7faf.apps.googleusercontent.com'
```

Comment:
I'm not too familiar with the API myself, but my hunch is that these constants should not be in the codebase itself, and rather should be in some `.env` file that's not a part of Git, so that random people can't misuse our API keys. We can talk about this more in-depth on Slack.

Reply:
Sure, just slack me when u get free time. Thanks!